src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -157,6 +158,22 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
         return mOpCredsIssuer.get();
     }
 
+    void SetDeviceAttestationDelegateBridge(DeviceAttestationDelegateBridge * deviceAttestationDelegateBridge)
+    {
+        mDeviceAttestationDelegateBridge = deviceAttestationDelegateBridge;
+    }
+
+    DeviceAttestationDelegateBridge * GetDeviceAttestationDelegateBridge() { return mDeviceAttestationDelegateBridge; }
+
+    void ClearDeviceAttestationDelegateBridge()
+    {
+        if (mDeviceAttestationDelegateBridge != nullptr)
+        {
+            delete mDeviceAttestationDelegateBridge;
```

Comment:
can we use some std::uniqe_ptr to handle lifetimes?

Reply:
> can we use some std::uniqe_ptr to handle lifetimes?

Each commissioning will set a new delegate and replace the last delegate, using unique_ptr does not seem to bring significant benefits.
Maybe I can add the call to delete `mDeviceAttestationDelegateBridge`  in `AndroidDeviceControllerWrapper::OnCommissioningComplete` ?